core/templates/services/sidebar-status.service.ts

```diff
@@ -29,20 +29,28 @@ export class SidebarStatusService {
   constructor(private wds: WindowDimensionsService) {}
   private pendingSidebarClick: boolean = false;
   private sidebarIsShown: boolean = false;
+  private _toggleSidebarEventEmitter: EventEmitter<void> = (
+    new EventEmitter());
+
+  get toggleSidebarEventEmitter(): EventEmitter<void> {
+    return this._toggleSidebarEventEmitter;
+  }
+
   private _onSideBarStatusUpdate: EventEmitter<boolean> = new EventEmitter();
 
   private _openSidebar(): void {
     if (this.wds.isWindowNarrow() && !this.sidebarIsShown) {
       this.sidebarIsShown = true;
       this.pendingSidebarClick = true;
-      this._onSideBarStatusUpdate.emit(true);
+      this.toggleSidebarEventEmitter.emit();
```

Comment:
make sure the sidebar icon in top navigation bar changes when you close it by swiping on a mobile device.

If that works, remove the _onSIdeBarStatusUpdate Event Emitter from the service and its subscription in TNB.

Reply:
Works fine removed this